.github/workflows/zulip-ci.yml

```diff
@@ -35,6 +29,18 @@ jobs:
             os: jammy
             legacy_client_interface: "6"
             server_version: refs/tags/6.0
+          # Debian 12 ships with Python 3.11.2.
+          - docker_image: zulip/ci:bookworm
+            name: Debian 12 (Python 3.11, backend + documentation)
+            os: bookworm
+            legacy_client_interface: "7"
+            server_version: refs/tags/7.0
+          # Ubuntu 24.04 ships with Python 3.12.3.
```

Comment:
Though, the Zulip workflow mentions [3.12.2](https://github.com/zulip/zulip/blob/8.3/.github/workflows/zulip-ci.yml#L57), the actual version used now seems to be 3.12.3.